src/components/implementation/no_interface/scheddev/fprr/sl_mod_fprr.c

```diff
@@ -35,6 +38,7 @@ void
 sl_mod_block(struct sl_thd_policy *t)
 {
 	ps_list_rem_d(t);
+	sl_timeout_mod_block(sl_mod_thd_get(t), 0);
```

Comment:
Yes, we do. (Except for AEPs as they're explicitly woken up using `asnd`s.). Your point is valid and I think I understand that now. This code should really be in `sl_thd_block()` and not here as we always use it's period as timeout and sl_thd has that information. Also, none of this is module specific, you're right. Thank you!